config/crd/patches/cainjection_in_idms.yaml

```diff
@@ -7,4 +7,4 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
-  name: idms.idmocp.redhat.com
+  name: freeipas.idmocp.redhat.com
```

Comment:
This should be  `idms.idmocp.redhat.com` instead of `freeipas.idmocp.redhat.com`.

Now it is fixed.